hadoop-ozone/dev-support/intellij/runConfigurations/OzoneFsShell.xml

```diff
@@ -0,0 +1,36 @@
+<!--
+   Licensed to the Apache Software Foundation (ASF) under one or more
+   contributor license agreements.  See the NOTICE file distributed with
+   this work for additional information regarding copyright ownership.
+   The ASF licenses this file to You under the Apache License, Version 2.0
+   (the "License"); you may not use this file except in compliance with
+   the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+-->
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="OzoneFsShell" type="Application" factoryName="Application">
+    <envs>
+      <env name="OZONE_CONF_DIR" value="$PROJECT_DIR$/hadoop-ozone/dev-support/intellij" />
+    </envs>
+    <option name="MAIN_CLASS_NAME" value="org.apache.hadoop.fs.ozone.OzoneFsShell" />
+    <module name="hadoop-ozone-filesystem" />
```

Comment:
```suggestion
    <module name="hadoop-ozone-tools" />
```

Reply:
I'm glad to see the explicit _commons-cli_ jar file included to be unnecessary - great!  Using the _hadoop-ozone-tools_ module instead of _hadoop-ozone-filesystem_ with the _OzoneFsShell_ main resolves our external jar issues, thanks for this **@elek**.  Much better.  Committing suggestions.